macros/sql_gen/gen_primary_metric_aggregate.sql

```diff
@@ -59,6 +62,18 @@
         sum({{ expression }})
 {%- endmacro -%}
 
+{% macro default__metric_median(expression) %}
+        median({{ expression }})
+{%- endmacro -%}
+
+{% macro bigquery__metric_median(expression) %}
+        any_value({{ expression }})
```

Comment:
If you ever need an implementation of `any_value` that will work across adapters, then you can use the following:

[`{{ dbt.any_value("expression") }}`](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros#any_value)

Of course this particular implementation is BigQuery-specific, so it doesn't need a cross-database macro like above.